getting-started/logging-out.rst

```diff
@@ -0,0 +1,19 @@
+###########
+Logging out
+###########
+
+Once you’re ready to take a break, log out from the Notebook Aspect to release computational resources to others.
+Don’t worry, all your files — and even the layout of tabs — are saved. When you log in later, you can pick right up.
+
+The best way to log out is by clicking on **File** → **Save All, Exit, and Log Out**.
```

Comment:
Very happy to see this advice! I thought I was supposed to do Kernel -> Log Out and I kept forgetting to do that.